browser/extensions/api/brave_shields_api_browsertest.cc

```diff
@@ -321,4 +331,16 @@ IN_PROC_BROWSER_TEST_F(BraveShieldsAPIBrowserTest,
   EXPECT_EQ(setting, CONTENT_SETTING_BLOCK);
 }
 
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
```

Comment:
nit: This test should be in new files like `brave_wallet_api_browsertest.cc`?

Reply:
Moved to `brave_wallet_api_browsertest.cc`. Good suggestion, thanks.